lib/ansible/playbook/play.py

```diff
@@ -58,7 +59,7 @@ def __init__(self, playbook, ds, basedir, vault_password=None):
 
         for x in ds.keys():
             if not x in Play.VALID_KEYS:
-                raise errors.AnsibleError("%s is not a legal parameter at this level in an Ansible Playbook" % x)
+                raise errors.AnsibleError("%s is not a legal parameter of an Ansible Play" % x)
```

Comment:
My mistake; didn't fully understand the context and previous line referenced "Playbook". Thanks for the clarification.